fluentforms/core/src/main/java/com/_4point/aem/fluentforms/impl/output/OutputServiceImpl.java

```diff
@@ -139,7 +141,8 @@ private Document generatePrintedOutput(String urlOrFileName, Document data, Prin
 	@Override
 	public GeneratePrintedOutputArgumentBuilder generatePrintedOutput() {
 		// TODO Auto-generated method stub
-		throw new UnsupportedOperationException("Not implemented yet.");
+		//throw new UnsupportedOperationException("Not implemented yet.");

```

Comment:
Can we remove the comments that are no longer current?  There's no reason to keep old code.  It is all stored in the repository if someone needs it.